### PR TITLE
Create key_tentpole_dates file

### PR DESCRIPTION
**Pull request by ryboland**
Track Key Tentpole dates to bring table into larger model

## Related Tickets & Documents
* https://mozilla-hub.atlassian.net/browse/AD-208

**Reviewer, please follow [this checklist](https://github.com/mozilla/bigquery-etl/blob/main/.github/reviewer_checklist.md)**

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AD-208)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv [hunk 1]
```diff
@@ -0,0 +1,113 @@
+Key Tentpole	Start Date	End Date
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv:1]: This needs to be a comma-delimited CSV

File: sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+New Year's  	January 1	January 1
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv:2]: Can you put all the dates in `YYYY-MM-DD` format?

File: sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Columbus Day	October 13, 2025	October 13, 2025
+Veterans Day	November 11, 2025	November 11, 2025
+Thanksgiving Day	November 27, 2025	November 27, 2025
+Christmas Day	December 25, 2025	December 25, 2025
+New Year's Day	January 1, 2026	January 1, 2026
+Birthday of Martin Luther King, Jr.	January 19, 2026	January 19, 2026
+Washington's Birthday	February 16, 2026	February 16, 2026
+Memorial Day	May 25, 2026	May 25, 2026
+Juneteenth	June 19, 2026	June 19, 2026
+Independence Day (observed) †	July 3, 2026	July 3, 2026
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv:64]: I would get rid of these special characters

**Review by curtismorales — CHANGES_REQUESTED** [review]
A couple quick changes and this will be ready to go

File: sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv (hunk 2 above)
Comment (ryboland) [reply sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv]: @curtismorales Can do!

**Review by curtismorales — APPROVED** [review]

**Pull request merged**